.github/workflows/ci-test.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         uses: snok/install-poetry@v1
 
       - name: Poetry install
-        run: poetry install
+        run: poetry install --with="dev,test" --extras="duckdb snowflake datafusion"
```

Comment:
annoyingly, `--with` takes commas but not spaces and `--extras` takes spaces but not commas